src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -26,20 +35,35 @@ namespace orbit_object_utils {
 
 namespace {
 
+[[nodiscard]] uint64_t ComputeRelativeAddress(
```

Comment:
I think the name is wrong. Isn't this computing the _Virtual_ Address, as per definition in https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#general-concepts?